bertopic/representation/_openai.py

```diff
@@ -5,8 +5,9 @@
 from scipy.sparse import csr_matrix
 from typing import Mapping, List, Tuple, Any, Union, Callable
 from bertopic.representation._base import BaseRepresentation
-from bertopic.representation._utils import retry_with_exponential_backoff, truncate_document
+from bertopic.representation._utils import retry_with_exponential_backoff, truncate_document, MyLogger
 
+logger = MyLogger("WARNING")
```

Comment:
I'm not sure whether importing the logger will work with the current logging procedure. My guess is that it will create a separate logger which does things twice. Also, the model should control the verbosity so that this warning is suppressed if needed by the user. 

I believe, but I will have to check, that the functionality of the logger will change in an open PR.

Reply:
That makes total sense. Since we want to leave verbosity controls to the model, I could use the logger instance from `topic_model.logger`. Wyt?

Reply:
Good thinking, that would things much easier.